guides/common/modules/con_organizations-and-locations-in-project.adoc

```diff
@@ -13,3 +22,37 @@ Locations typically represent physical locations, such as countries or cities.
 +
 By creating locations, you can define geographical sites where hosts are located.
 For example, this is useful in environments with multiple data centers.
++
+You can use locations to map the network infrastructure to prevent incorrect host placement or configuration.
+While you cannot assign a subnet, domain, or compute resources directly to a {SmartProxyServer}, you can assign them to a location.
++
+Unlike organizations, locations can have a hierarchical structure.
+
+{ProjectServer} defines all locations and organizations.
+Each respective {Project} {SmartProxyServer} synchronizes content and handles configuration of systems in a different location.
+
+The main {ProjectServer} retains the management function, while the content and configuration is synchronized between the main {ProjectServer} and a {Project} {SmartProxyServer} assigned to certain locations.
+
+
+.Example of using organizations and locations in {Project}
+====
+The structure of a multi-national company includes the Finance, Marketing, and Sales departments. The company operates across United States, United Kingdom, and Japan.
+
+The system administrator creates the following _organizations_ on their {ProjectServer}:
```

Comment:
I am unsure if emphasis is needed. Your call.

Reply:
Dropped.